department-of-veterans-affairs/vets-website
# Update copy

## Summary
On the Pension Benefits application(21P-527EZ),
Update copy

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Select 'Start your application without signing in'
3. Verify the copy changes in the acceptance criteria

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70331)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots
_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

### Desktop
| Military Service Number | Other Service Names | 
| ------ | ------ |
|![localhost_3001_pension_application_527EZ_military_history (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/860a5623-abb0-4d14-825f-ea753b501515)|![localhost_3001_pension_application_527EZ_military_history](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/2975388b-5b6e-4f65-b429-d5bbbf606b20)



## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70331)

### Quality Assurance & Testing

### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
